src/components/CompanyAndJobTitle/CompanyAndJobTitleWrapper.js

```diff
@@ -14,6 +21,17 @@ const CompanyAndJobTitleWrapper = ({
   tabType,
 }) => {
   const helmetTitle = `${pageName} ${tabTypeTranslation[tabType]}`;
+  const tabLinkOptions = useMemo(
+    () =>
+      compose(
+        map(([type, label]) => ({
+          label,
+          to: `/${pageTypeURLMap[pageType]}/${pageName}/${tabTypeURLMap[type]}`,
```

Comment:
我想到我們好像沒有討論`encodeURICompoent(pageName)`，這個是不是應該在`to`這裡就先做？

Reply:
我覺的可以用

generatePath("/:company", {company: "你好"})
```suggestion
          to: generatePath("/:pageTypeURL/:pageName/:tabTypeURL", {
            pageTypeURL: pageTypeURLMap[pageType],
            pageName,
            tabTypeURL: tabTypeURLMap[type],
          }),
```

這個的好處是：

```js
generatePath("/:name/:type", {name: "GoodJob 好工作", type: "site"})
// ==> '/GoodJob%20%E5%A5%BD%E5%B7%A5%E4%BD%9C/site'
```

它會自動 encode，另外就是如果想要抽成

```js
const some = ({ pageType, pageName, type }) => generatePath("/:pageTypeURL/:pageName/:tabTypeURL", {
  pageTypeURL: pageTypeURLMap[pageType],
  pageName,
  tabTypeURL: tabTypeURLMap[type],
})
```

也是可行